front/src/styles/scss/applications/stdcm/_card.scss

```diff
@@ -89,7 +92,9 @@
     }
 
     .feed-back {
-      padding: 0;
+      --field-wrapper-padding-bottom: 9px;
```

Comment:
Why do we need to override the osrd-ui styles here?

Reply:
It happens because we modified the input paddings in this form, and the tooltip offset is calculated based on the padding values.

Reply:
Alright. I'm not in love with the way this is done, but I've dragged this PR for too long already, so let's leave it as-is.